connection.go

```diff
@@ -0,0 +1,100 @@
+package ts3
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"net"
+	"strings"
+	"time"
+
+	"golang.org/x/crypto/ssh"
+)
+
+const (
+	// DefaultPort is the default TeamSpeak 3 ServerQuery port.
+	DefaultPort = 10011
+
+	// DefaultSSHPort is the default TeamSpeak 3 ServerQuery SSH port.
+	DefaultSSHPort = 10022
+)
+
+// legacyConnection is an insecure TCP connection.
+type legacyConnection struct{ net.Conn }
+
+// Connect connects to the address with the given timeout.
+func (c *legacyConnection) Connect(addr string, timeout time.Duration) error {
+	if !strings.Contains(addr, ":") {
+		addr = fmt.Sprintf("%v:%v", addr, DefaultPort)
+	}
```

Comment:
This method will fail for ipv6 addresses. For robustness, you probably want to use `net.SplitHostPort` and `net.JoinHostPort` and look out for `&net.AddrError{Err:"missing port in address"}` errors to handle cases where no port is provided and maybe also look for `&net.AddrError{Err:"too many colons in address"}` for cases where ipv6 is passed without the surrounding square brackets.

Reply:
Yes, I didn't even think about that and just copied it from `NewClient`. 
I don't think `too many colons in address` has to be handled seperately so I just return the error.